dimod/generators/random.py

```diff
@@ -500,3 +500,61 @@ def doped(p: float, graph: GraphLike,
         bqm.add_interaction(u, v, J)
 
     return bqm
+
+
+
+@graph_argument('graph')
+def power_r(r: int, graph: GraphLike,
+          seed: Optional[int] = None) -> BinaryQuadraticModel:
+    """Generate an Ising model for a POWERr problem.
+
+    In POWERr problems all linear biases are zero and quadratic values are integers 
```

Comment:
Is there a citation for this problem class?

Reply:
not yet!